### ISSUE TITLE
Music does not play

### ISSUE THREAD

**Issue by QEDan** [tilk/cdlc_player#3]
This is really great. Thank you for making this.

When I play a song in the app, the only audio is the metronome click. The music does not play. But, I can see the 3D tab for the song scrolling. I'm using LineageOS 18.1 on a Pixel 3a phone. I have version 0.0.5 of the app from F-droid.

**Comment by tilk** [comment]
No, it doesn't. I should have mentioned this in the README. It's not implemented because (a) it's probably hard (the sound is encoded in some weird Wwise format), (b) implementing this would possibly make this project more of a target for lawyers and stuff.

**Comment by QEDan** [comment]
I see. That's a bit disappointing, but I understand the reasons. Thanks for responding. 

I do think it would be helpful to explain this in the README and the F-droid description to help set expectations for users.

**Comment by ghost** [comment]
I know it's been ages but I would be delighted if you can test my very alpha support for just that! https://github.com/yassineim/cdlc_player/releases/tag/audio-v4

Also raised a PR which I hope will get merged despite the prebuilt *.so, the thing being that building those blobs is a big headache to document, not that it's hard or anything, just tedious.

**Comment by QEDan** [comment]
@yassineim It works! Thank you so much for adding that. I will report any problems I encounter.

**Comment by ghost** [comment]
@QEDan @tilk 

I added a lot of improvements that reflect both to performance and the resulting apk size. I hope you can check.

https://github.com/yassineim/cdlc_player/releases/tag/audio-v4

**Comment by sanicki** [comment]
Will the audio-playing version be added to F-Droid?
